driver/features.go

```diff
@@ -7,3 +7,5 @@ const DockerExporter Feature = "Docker exporter"
 
 const CacheExport Feature = "Cache export"
 const MultiPlatform Feature = "Multi-platform build"
+
+const DefaultLoad Feature = "Automatically load images"
```

Comment:
```suggestion
const DefaultLoad Feature = "Automatically load images to docker store"
```

@dvdksn for copy :pray: